libs/uw-imap/patches/004-fix-linking.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/src/osdep/unix/Makefile b/src/osdep/unix/Makefile
+index 78913ac..9eb9c9c 100644
+--- a/src/osdep/unix/Makefile
++++ b/src/osdep/unix/Makefile
+@@ -962,21 +962,24 @@ os_sol.h:
+ 
+ # Once-only environment setup
+ 
+-once:	onceenv ckp$(PASSWDTYPE) ssl$(SSLTYPE) osdep.c
++once:	ssl$(SSLTYPE) onceenv ckp$(PASSWDTYPE) osdep.c
+ 
+ onceenv:
```

Comment:
NAK.  Do __not__ put this stuff into files.  There are variables for passing ALL of this stuff.

I know the original sources do that, but no...  This breaks outside the LEDE model.

Just do `@touch once` just before the `$(MAKE)`, or (better) write a clean patch to get rid of all this bogosity and remove all of the `$(cat FILE)` stuff.

Reply:
yes, the problem is that anyway, all the Makefile is full of cat, or are you referring just for the patchig ?

Reply:
Unfortunately, all of it.

Reply:
so I prepared the files, but can't get the values from them inside the recipe printed on screen while doing the $(CC), is not ok to just print the arguments before compiling ?
the problem is that the other recipe are preparing the files that contain arguments so only the final recipe uses all the arguments, taken from the final files
didn't find enough documentation on how to read the file inside the recipe and have it assigned to a variable, but I could print the content

Reply:
an alternative would be to make a new Makefile just for one target (slx) and have all the variables combined before entering the recipe

Reply:
> didn't find enough documentation on how to read the file inside the recipe and have it assigned to a variable, but I could print the content

No, I was suggesting doing away with the files altogether and passing everything through the command line as variable assignments.